src/modules/netinstall/README.md

```diff
@@ -0,0 +1,28 @@
+# Netinstall module
+
+The netinstall module allows distribution maintainers to ship minimal ISOs with only a basic set of preinstall packages. At installation time, the user is presented with the choice to install groups of packages from a predefined list.
+
+Calamares will then invoke the correct backend to install the packages.
+
+## How are packages configured?
+The *netinstall.conf* file should have this format:
+
+    ----
+    groupsUrl: <URL to YAML file>
```

Comment:
My reasoning was that if the file is local, then changing even the name of one package or adding a new one requires a rebuild for the distro maintainer. I imagine both actions will be performed pretty often.

However I am thinking that reading from a remote location has potential security issues; we would need, at least, to avoid displaying any string above a certain size, and make sure the package names are not malformed (although the threat of remote command execution may be a bit far-fetched). Thoughts? :)